_includes/sc/syllabus.html

```diff
@@ -35,16 +35,15 @@ <h3 id="syllabus-r">R para análisis científicos reproducibles</h3>
     <h3 id="syllabus-git">El control de versiones con Git</h3>
     <ul>
       <li>Creando un repositorio</li>
-      <li>Rastreando Cambios: <code>add</code>, <code>commit</code>, ...</li>
-      <li>Explorando el **History**: <code>status</code>, <code>diff</code>, ...</li>
-      <li>Ignorando cosas</li>
+      <li>Rastreando cambios: <code>add</code>, <code>commit</code>, ...</li>
+      <li>Explorando el **historial**: <code>status</code>, <code>diff</code>, ...</li>
+      <li>Ignorando cambios</li>
```

Comment:
The episodes is "ignorando cosas", as it is when itroduces gitignore. They are not really changes, but more than that (you can ignore untrack files, or ignore new changes on previously tracked ones).
```suggestion
      <li>Ignorando cosas</li>
```